src/CVSelect/ContentViewList.js

```diff
@@ -39,7 +39,19 @@ export default function ContentViewList() {
     window.location.assign(`/content-view/${id}`);
   };
 
-  const handleOnSearchChange = () => {};
+  const handleOnSearchChange = (val) => {
+    if (val === "") {
+      contentViews.forEach((cv) => {
+        document.getElementById(cv.name.toLowerCase()).style.display = "block";
```

Comment:
There's a good chance that these might end up being flexboxes. Let's create a class that hides elements, and add/remove that instead.